azure-pipelines-official.yml

```diff
@@ -82,22 +82,22 @@ stages:
         nugetConfigPath: 'NuGet.config'
         restoreDirectory: '$(Build.SourcesDirectory)\.packages'
 
-    - task: ms-vseng.MicroBuildTasks.30666190-6959-11e5-9f96-f56098202fef.MicroBuildSigningPlugin@1
+    - task: MicroBuildSigningPlugin@2
       inputs:
         signType: $(SignType)
         zipSources: false
       condition: and(succeeded(), in(variables['SignType'], 'test', 'real'))
 
-    - task: ms-vseng.MicroBuildTasks.965C8DC6-1483-45C9-B384-5AC75DA1F1A4.MicroBuildOptProfPlugin@1
+    - task: MicroBuildOptProfPlugin@5
       inputs:
-        skipRunOptimize: true
+        ShouldSkipOptimize: true
       displayName: 'Install OptProf Plugin'
 
     # Required by MicroBuildBuildVSBootstrapper
```

Comment:
@genlu With the centralized OptProf training, do we still need to build the bootstrapper? I though all this will go away once we switch.

Reply:
No we won't. But we haven't got to remove everything related to that yet.